apis/r/NEWS.md

```diff
@@ -83,6 +83,16 @@ Effectively, this stores the data on disk in column-major order. However, the `t
 - Added `with_allocation_size_preference()` helper to temporarily set the allocation size preference for testing.
 - Tests were added to verify the internal `dgtmatrix_to_dataframe()` will error out if an input list contains non-layerable matrices.
 
```

Comment:
Thanks for adding! But this whole block should be moved to the top of `NEWS.md`, and `# tiledbsc (development version)` should be replaced with `# tiledbsoma 0.1.12` (like [so](https://gist.github.com/aaronwolen/4962f6cfc25a362cadd1175390042c4b)).